doc/api/repl.md

```diff
@@ -444,7 +444,7 @@ added: v2.0.0
 deprecated: v3.0.0
 -->
 
-   Stability: 0 - Deprecated: Use `NODE_REPL_HISTORY` instead.
+> Stability: 0 - Deprecated: Use `NODE_REPL_HISTORY` instead.
```

Comment:
Niiice. This fixed an actual error: https://nodejs.org/api/repl.html#repl_node_repl_history_file — that line had three spaces instead of four =).